tiledb/cloud/_results/results.py

```diff
@@ -68,14 +78,54 @@ def to_stored_param(self) -> stored_params.StoredParam:
             task_id=self.task_id,
         )
 
-    # The HTTP content of the body that was returned.
-    body: bytes
-    # The server-generated UUID of the task.
-    task_id: Optional[uuid.UUID]
-    # The decoder that was used to decode the results.
-    decoder: decoders.AbstractDecoder[_T]
-    # True if the results were stored, false otherwise.
-    results_stored: bool
+    task_id = attr.ib(type=Optional[uuid.UUID])
```

Comment:
Mostly out of curiosity, why are you using `.ib` rather than `field` + type annotation here?

Reply:
Mostly because at the time, `attr.ib` was the primary API for attrs. I switched it over to the modern version.

Reply:
Alright, your call